hle-survey/transformation/src/test/scala/org/broadinstitute/monster/dap/HealthTransformationsSpec.scala

```diff
@@ -697,6 +697,124 @@ class HealthTransformationsSpec extends AnyFlatSpec with Matchers {
     output should contain theSameElementsAs truth
   }
 
+  it should "correctly map congenital GI disorders when values are defined" in {
+    val multipleCongenitalSkinDisorder = Map[String, Array[String]](
```

Comment:
also doesn't really matter but this says "Skin" in the variable name instead of "GI" or something